CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+
+# Email and Chat
+
+- Email: [operator-framework](https://groups.google.com/forum/#!forum/operator-framework)  
+
+## Setting up a dev environment
+
+- Fork the repository on GitHub
+- Clone the repository in your go path.
+- Get access to an Openshift cluster. The marketplace-operator is currently supported on Openshift 4.0. You can find instructions on installing an Openshift cluster [here](https://github.com/openshift/installer).
+- Turn off the Cluster-Service-Operator (CVO). The CVO manages the lifecycle of the marketplace-operator along with other internal Openshift components. Once the CVO is truned off, the marketplace-operator deployment can be deleted. 
+    `oc scale --replicas 0 -n openshift-cluster-version deployments/cluster-version-operator`
+- Delete the marketplace-operator deployment `oc delete deployment openshift-marketplace -n marketplace-operator`
+- Install [Operator-SDK](https://github.com/operator-framework/operator-sdk).
+- `cd` into the marketplace-operator repository and `dep ensure` to download all dependencies.
+- `operator-sdk up local --namespace=openshift-marketplace --kubeconfig=<path-to-kubeconfig-file>` to complie the marketplace-operator and start the operator in you dev environment.      
+
+## Reporting bugs and creating issues
+
+Reporting bugs is one of the best ways to contribute. If any part of the operator-marketplace project has bugs or documentation mistakes, please let us know by [opening an issue](https://github.com/operator-framework/operator-marketplace/issues/new)
+
+## Contribution flow
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Create a topic branch from where to base the contribution. This is usually master.
+- Make commits of logical units.
+- Make sure commit messages are in the proper format (see below).
+- Push changes in a topic branch to a personal fork of the repository.
+- Submit a pull request to operator-framework/operator-marketplace.
+- The PR must receive a LGTM from two maintainers of the operator-marketplace project.
+
+Thanks for contributing!
+
+### Format of the commit message
+
+We follow a rough convention for commit messages that is designed to answer two
+questions: what changed and why. The subject line should feature the what and
+the body of the commit should describe the why.
+
+```
+bug 1683422: [csc] Make CLI output less verbose
+
+- Fixes #116
+- Removed TARGETNAMESPACE and PACKAGES fields from output
+- Modified catalogsourceconfig.crd to remove fields from `additionalPrinterColumns`
+
+```
+
+The format can be described more formally as follows:
+
+```
+<subsystem>: <what changed>
+<BLANK LINE>
+<why this change was made>
+<BLANK LINE>
+<footer>
+```
+
+The first line is the subject and should be no longer than 50 characters, the second line is always blank, and other lines should be wrapped at 72 characters. This allows the message to be easier to read on GitHub as well as in various git tools.
```

Comment:
Are you following the 72 character limit in your commit message? :-)

Reply:
@aravindhp For the other PRs yes. For this one I edited my message now and it's following the convention :)

Reply:
We can go with 80 characters instead of 72.